colossalai/inference/modeling/models/nopadding_baichuan.py

```diff
@@ -5,15 +5,61 @@
 import torch.nn as nn
 
 from colossalai.inference.flash_decoding_utils import FDIntermTensors
-from colossalai.inference.modeling.models.nopadding_llama import NopadLlamaAttention
 from colossalai.kernel.kernel_loader import InferenceOpsLoader
+from colossalai.kernel.triton import (
+    context_attention_unpadded,
+    copy_k_to_blocked_cache,
+    decoding_fused_rotary_embedding,
+    flash_decoding_attention,
+    rms_layernorm,
+    rotary_embedding,
+)
 from colossalai.logging import get_dist_logger
 
+logger = get_dist_logger(__name__)
+
+try:
+    from flash_attn import flash_attn_varlen_func
+
+    use_flash_attn2 = True
+except ImportError:
+    use_flash_attn2 = False
+    logger.warning(f"flash_attn2 has not been installed yet, we will use triton flash attn instead.")
+
 inference_ops = InferenceOpsLoader().load()
 
 logger = get_dist_logger(__name__)
 
 
+def baichuan_rmsnorm_forward(
+    self,
+    hidden_states: torch.Tensor,
+    norm_output: torch.Tensor,
+    residual: torch.Tensor = None,
+    use_cuda_kernel: bool = True,
+):
+    # Used to address the issue of inconsistent epsilon variable names in baichuan2 7b and 13b.
+    if hasattr(self, "variance_epsilon"):
```

Comment:
What I'm thinking is that if Baichuan releases a 70B model and they change the variable names again.

Reply:
Hence, at this moment, only two possibilities exist (7B/13B). Please comply with Python logic and styling. Change to `elif` when there have already been more options.

Reply:
Alternatively, you could comply with Python logic and styling by adding `else` to remind the user unexpected attribute found. **The idea here is to handle all the conditions** when we use if-else blocks or if-elif-else blocks, instead of allowing some condition fleeing away and cause unexpected error.

Reply:
OK